{
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unaffected",
                    "product": "389-ds-base",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "1.4"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "<p>A vulnerability was discovered in the 389 Directory Server that allows an unauthenticated attacker with network access to the LDAP port to cause a denial of service. The denial of service is triggered by a single message sent over a TCP connection, no bind or other authentication is required. The message triggers a segmentation fault that results in slapd crashing.</p>"
                        }
                    ],
                    "value": "A vulnerability was discovered in the 389 Directory Server that allows an unauthenticated attacker with network access to the LDAP port to cause a denial of service. The denial of service is triggered by a single message sent over a TCP connection, no bind or other authentication is required. The message triggers a segmentation fault that results in slapd crashing.\n\n"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Uncontrolled Resource Consumption",
                            "lang": "en"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat",
                "dateUpdated": "2023-01-04T14:21:25.880Z"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2055815"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://access.redhat.com/security/cve/CVE-2022-0918"
                },
                {
                    "tags": [
                        "issue-tracking"
                    ],
                    "url": "https://github.com/389ds/389-ds-base/issues/5242"
                },
                {
                    "url": "https://lists.debian.org/debian-lts-announce/2023/04/msg00026.html"
                }
            ],
            "source": {
                "discovery": "UNKNOWN"
            },
            "x_generator": {
                "engine": "Vulnogram 0.1.0-dev"
            },
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2022-0918",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "389-ds-base",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "1.4"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "A vulnerability was discovered in the 389 Directory Server that allows an unauthenticated attacker with network access to the LDAP port to cause a denial of service. The denial of service is triggered by a single message sent over a TCP connection, no bind or other authentication is required. The message triggers a segmentation fault that results in slapd crashing."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "Uncontrolled Resource Consumption"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=2055815",
                            "refsource": "MISC",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2055815"
                        },
                        {
                            "name": "https://access.redhat.com/security/cve/CVE-2022-0918",
                            "refsource": "MISC",
                            "url": "https://access.redhat.com/security/cve/CVE-2022-0918"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "assignerShortName": "redhat",
        "cveId": "CVE-2022-0918",
        "datePublished": "2022-03-16T14:04:23",
        "dateReserved": "2022-03-10T00:00:00",
        "dateUpdated": "2023-01-04T14:21:25.880Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}